mscore/revision.h

```diff
@@ -1 +1 @@
-3543170
+e122e7c
```

Comment:
Is this file supposed to check-in???

Reply:
Ooops, I'll remove that.

Reply:
better install that hook to prevent this

Reply:
Check https://musescore.org/en/handbook/developers-handbook/finding-your-way-around/git-workflow#Summary step 4: 

    In your clone directory, copy the file build/git/hooks/post-checkout to the directory .git/hooks in order for mscore/revision.h to be maintained automatically by git. See build/git/hooks/README for details. Note that the .git directory is hidden.

Reply:
Thanks, I'll do that tomorrow.

Reply:
Done. Should I squash my commits for this to take effect?

Reply:
Yes, I think so